src/dsp_tools/commands/xmlupload/models/ingest.py

```diff
@@ -109,20 +107,15 @@ def _ingest(self, filepath: Path) -> IngestResponse:
                 elif res.status_code == STATUS_UNAUTHORIZED:
                     raise BadCredentialsError("Bad credentials")
                 else:
-                    user_msg = f"{err} See {WARNINGS_SAVEPATH} for more information."
-                    print(user_msg)
```

Comment:
printing interfers with the progress bar. The error message can be retrieved from the logs and from the `warnings.log`